packages/p2p/src/pages/my-profile/payment-methods/payment-methods-list/edit-payment-method-form.jsx

```diff
@@ -156,4 +172,4 @@ const EditPaymentMethodForm = () => {
     );
 };
 
-export default observer(EditPaymentMethodForm);
+export default EditPaymentMethodForm;
```

Comment:
Wrapping this component with observer caused the console logging error issue, which is caused by updating the payment method, which is then updating any observables inside this component whilst it is unmounting. 

We don't need to have observer wrapping this component since it doesn't need to listen to any changes except if there are errors whilst updating the payment method